addons/website_blog/models/website_blog.py

```diff
@@ -128,15 +128,11 @@ def _compute_website_url(self):
             blog_post.website_url = "/blog/%s/%s" % (slug(blog_post.blog_id), slug(blog_post))
 
     def _default_content(self):
-        return f'''
-            <section class="s_text_block pt16 pb16" data-snippet="s_text_block">
-                <div class="o_container_small">
-                    <p>
-                        {_("Start writing here...")}
-                    </p>
-                </div>
-            </section>
-        '''
+        return self.env['ir.ui.view']._render_template('website.s_text_block', {
```

Comment:
@qsm-odoo 
 I don't see how to use it in demo data, to avoid duplication.

Reply:
`ref(view_xml_id)._render(values)` or something like that should work, but I don't really know for sure without grepping the code, searching and testing

Reply:
I don't see how to mix an xml demo data with python code (trying the eval doesn't seem the right way).

Reply:
Not commenting XML demo data here so not sure what you are talking about... For demo data, you will indeed have to duplicate the snippet XML I think. But here you can render the template for the field's default value... like the code you are commenting, is your code or my suggestion not working?

Reply:
It works, I just want to avoid duplication in demo data ...